add support for monitoring LVM caches

anarcat opened the pull request:
This is basically a copy-paste of the snapshots code, and seems to work on my end.

Closes: #158

SuperQ reviewed (changes requested):
Can you rebase this with the `-a` flag and include a `display_usage()` update?

anarcat committed:
add -a usage (#160)

Signed-off-by: Antoine Beaupré <anarcat@debian.org>

anarcat commented:
this has been rebased on top of #161 and #160 (master).

anarcat commented:
@SuperQ ping

anarcat committed:
add support for monitoring LVM caches

This is basically a copy-paste of the snapshots code, and seems to
work on my end.

Closes: #158

Signed-off-by: Antoine Beaupré <anarcat@debian.org>

anarcat commented:
@SuperQ i can't seem to be able to tell github that your "requested changes" have been answered here, i rerolled on top of master too.

SuperQ reviewed (approved):
Thanks
